staging/src/k8s.io/apiserver/pkg/util/webhook/webhook_test.go

```diff
@@ -346,18 +348,6 @@ func TestTLSConfig(t *testing.T) {
 			serverCert: serverCert, serverKey: serverKey, serverCA: invalidCert,
 			errRegex: errBadCertificate,
 		},
-		{
-			test:       "invalid client certificate",
-			clientCert: invalidCert, clientKey: clientKey, clientCA: caCert,
-			serverCert: serverCert, serverKey: serverKey, serverCA: caCert,
-			errRegex: "tls: failed to find any PEM data in certificate input",
-		},
-		{
-			test:       "invalid client key",
-			clientCert: clientCert, clientKey: invalidCert, clientCA: caCert,
-			serverCert: serverCert, serverKey: serverKey, serverCA: caCert,
-			errRegex: "tls: failed to find any PEM data in key input",
-		},
```

Comment:
The following test cases are not valid anymore as the test fails in the [LoadKubeconfig ](https://github.com/kubernetes/kubernetes/blob/0c645922edcc06adff43c70c02fb56751364bbb5/staging/src/k8s.io/apiserver/pkg/util/webhook/webhook_test.go#L468)step itself.